package/MDAnalysis/lib/src/dimensionality_reduction/stochasticproxembed.h

```diff
@@ -1,27 +0,0 @@
-int trmIndex(int, int);
```

Comment:
Everything still compiles without it. I also think this has been moved to the `analysis/encore/dimensionality_reduction/spe.h` in the original encore PR where we asked that the cython sources are all moved to the same location where the final shared object file will be.